week15-18-05-11.md

```diff
@@ -0,0 +1,187 @@
+
+### Remove unnecessary Model conformance on Authenticatable
+[vapor/auth#35](https://github.com/vapor/auth/pull/35)
+
+
+## Tagged releases
+- [Vapor 3.0.2](https://github.com/vapor/vapor/releases/tag/3.0.2)
+- [HTTP 3.0.5](https://github.com/vapor/http/releases/tag/3.0.5)
+- [Redis 3.0.0 RC 2.6](https://github.com/vapor/redis/releases/tag/3.0.0-rc.2.6)
+- [Auth 2.0.0 RC4](https://github.com/vapor/auth/releases/tag/2.0.0-rc.4)
+- [URL-Encoded Form 1.0.3](https://github.com/vapor/url-encoded-form/releases/tag/1.0.3)
+- [WebSocket 1.0.1](https://github.com/vapor/websocket/releases/tag/1.0.1)
+- [Crypto 3.1.2](https://github.com/vapor/crypto/releases/tag/3.1.2)
+- [PostgreSQL 1.0.0 RC 2.2](https://github.com/vapor/postgresql/releases/tag/1.0.0-rc.2.2)
+
+## Articles
+### Vapor 3.0.0 release article
+Brought to you by Vapor's development team
+[Vapor 3 Release](https://medium.com/@codevapor/vapor-3-0-0-released-8356fa619a5d)
+
+### Server Side Swift frameworks comparison
+A bit old, given the fact everything changes really fast (Feb)
+
+[Server Side Swift Frameworks Comparison](https://www.netguru.co/codestories/server-side-swift-frameworks-comparison)
+
+### Transforming from Vapor 2 to Vapor 3
+Brought to you by [@calebkleveter](https://github.com/calebkleveter)
+
+[From Vapor 2 to Vapor 3](https://www.skelpo.com/blog/vapor2-to-vapor3/)
+
+### CRUD using Leaf
+Brought to you by [@MartinLasek](https://github.com/MartinLasek)
+
+[CRUD using Leaf](https://medium.com/@martinlasek/tutorial-how-to-write-crud-using-leaf-4d7dbae0979c)
+
+### Routing and Fluent
+Brought to you by [@calebkleveter](https://github.com/calebkleveter)
+[Routing and Fluent](https://medium.com/@caleb.kleveter/diving-into-vapor-part-3-introduction-to-routing-and-fluent-in-vapor-3-221d209f1fec)
+
+### Deploying Vapor 3 on Cloud Foundry
+Brought to you by [@idev4u](https://github.com/idev4u)
+[Deplying Vapor 3 in Cloud Foundry](https://medium.com/@Soulfected/vapor-3-on-cloud-foundry-e1861ef088d0?source=linkShare-33b4a6236f90-1525563379)
+
+### HTTP Basic Auth with Vapor 3
+Brought to you by [@anapaix](https://github.com/JoeyBodnar)
+[HTTP Basic Auth with Vapor 3](https://www.vaporforums.io/thread/41)
+
+### Getting started with Vapor 3 (Updated)
+Brought to you by [Paul Hudson](https://www.hackingwithswift.com/about)
+[Getting started with Vapor 3](https://www.hackingwithswift.com/articles/67/get-started-with-vapor-3-for-free)
+
+## Talks
+
+More talk coming soon
+
+## Want to know more or get inspired?
+- Check out what we are working on now. You might see something you can collaborate with [Issues](https://github.com/search?q=org%3Avapor+is%3Aissue+is%3Aopen+)
+- Check out more Vapor awesomeness [Awesome Vapor](https://github.com/Cellane/awesome-vapor)
+- Some more Vapor learning material [Vapor School](https://github.com/vaporberlin/vaporschool)
+- Great resources from our community [Vapor Forum](https://www.vaporforums.io)
```

Comment:
May also want to link https://forums.swift.org/c/related-projects/vapor

Checking with the owner of vaporforums.io to see what their plans are